serverlessworkflow/antora.yml

```
@@ -36,7 +36,7 @@ asciidoc:
     docker_min_version: 20.10.7
     docker_compose_min_version: 1.27.2
     # only used in downstream
-    operator_version: main
+    operator_version: 1.32.0
```

```suggestion
    operator_version: v1.32.0
```

See https://github.com/kiegroup/kogito-serverless-operator/releases